src/ResourceManager/Consumption/Commands.Consumption/help/Get-AzureRmConsumptionBudget.md

```diff
@@ -0,0 +1,144 @@
+---
+external help file: Microsoft.Azure.Commands.Consumption.dll-Help.xml
+Module Name: AzureRM.Consumption
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.consumption/get-azurermconsumptionbudget
+schema: 2.0.0
+---
+
+# Get-AzureRmConsumptionBudget
+
+## SYNOPSIS
+Get a list of budgets in either a subscription or a resource group.
+
+## SYNTAX
+
+```
+Get-AzureRmConsumptionBudget [-DefaultProfile <IAzureContextContainer>] [-ResourceGroupName <String>]
+ [-Name <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The Get-AzureRmConsumptionBudget cmdlet gets a list of budgets in either a subscription or a resource group.
+
+## EXAMPLES
+
+### Example 1: Get a list of budgets at subscription level
+```powershell
+PS C:\> Get-AzureRmConsumptionBudget
```

Comment:
Can you fix the formating for the examples? If you go look at the md file "view" option, you'll see that the formatting is off.

Reply:
Formatting was fixed across help files.